### ISSUE TITLE
Need to provision windows server 2012 R2 using knife oci

### ISSUE THREAD

**Issue by akashsarpate** [oracle/knife-oci#56]
We have to provision windows server 2012 R2 using knife oci.
Can we achieve this?
Please provide us knife oci command to provision windows server,

**Comment by briangustafson** [comment]
Window instances are not currently supported by knife-oci. The issue is that the Windows instances must first have WinRM manually enabled. However, once that is done you should be able to bootstrap using "knife bootstrap windows winrm".

**Comment by akashsarpate** [comment]
I am trying to bootstrap 
knife bootstrap windows winrm xx.xx.xx.xx --winrm-user opc --winrm-password 'abcd' --node-name winr12 and getting below error:
Creating new client for win
Creating new node for win

Waiting for remote response before bootstrap.ERROR: Failed to authenticate to xx.xx.xx.xx as opc
Response: WinRM::WinRMAuthorizationError
Hint: Make sure to prefix domain usernames with the correct domain name.
Hint: Local user names should be prefixed with computer name or IP address.
EXAMPLE: my_domain\user_namer

**Comment by briangustafson** [comment]
I've verified that I can bootstrap a Windows instance using image "Windows-Server-2012-R2-Standard-Edition-VM-2017.07.25-0". There are a number of manual steps - here are a few things to look for:

- Ensure that your security list is allowing TCP on ports 3389 and 5985.
- Ensure that WinRM is running on your instance. (After launching a new instance, I had to first set 
 the registry key LocalAccountTokenFilterPolicy under HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System, then run "winrm quickconfig".)
- Ensure that the Windows firewall isn't blocking WinRM.

Also, the winrm-user name that I used includes the compute name, such as "mywindowsinstance\opc".
